test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -36,6 +36,36 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& i
         input->mutable_config()->mutable_typed_config()->set_type_url(
             absl::StrCat("type.googleapis.com/",
                          factory->createEmptyConfigProto()->GetDescriptor()->full_name()));
+
+        // For fuzzing proto data, guide the mutator to useful 'Any' types half
+        // the time. The other half the time, let the fuzzing engine choose
+        // any message to serialize.
+        if (seed % 2 == 0 && input->data().has_proto_body()) {
+          // These types are request/response from the test Bookstore service
+          // for the gRPC Transcoding filter.
+          static const std::vector<std::string> expected_types = {
```

Comment:
Could the definition of these expected types be moved to `uber_per_filter.h`?

Reply:
I moved it to a function, is this what you mean?

It's difficult to just place the static vector in `uber_filter.h` unless it's scoped at the namespace level.